Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCollectionViewCell.swift

```diff
@@ -5,61 +5,74 @@
 import StreamChat
 import UIKit
 
+/// A View that is embed inside `UICollectionViewCell`  which shows information about user which we want to tag in suggestions
 public typealias ChatMessageComposerMentionCellView = _ChatMessageComposerMentionCellView<NoExtraData>
 
+/// A View that is embed inside `UICollectionViewCell`  which shows information about user which we want to tag in suggestions
 open class _ChatMessageComposerMentionCellView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
     // MARK: Properties
 
+    /// Content of the cell
+    /// This is tuple which contains:
+    /// - title: By default name of user
+    /// - subtitle: By default tag of user
+    /// - imageURL: URL of image for the user
+    /// - isUserOnline: Indicates if user is online to set onlineIndicator visible
     open var content: (title: String, subtitle: String, imageURL: URL?, isUserOnline: Bool)? {
         didSet {
             updateContentIfNeeded()
         }
     }
```

Comment:
One beautiful day when we will have time, this will become:
```
@Content title: String?
@Content subtitle: String?
@Content imageURL: URL?
@Content isUserOnline: Bool
```